packages/appeals-service-api/src/db/schema.prisma

```diff
@@ -266,3 +270,130 @@ model ServiceUser {
+  uploadDevelopmentPlanPolicies                   Boolean?
+  uploadOtherPolicies                             Boolean?
+  infrastructureLevy                              Boolean?
+  uploadInfrastructureLevy                        Boolean?
+  infrastructureLevyAdopted                       Boolean?
+  infrastructureLevyAdoptedDate                   DateTime?
+  infrastructureLevyExpectedDate                  DateTime?
+  uploadLettersInterestedParties                  Boolean?
+  treePreservationOrder                           Boolean?
+  uploadTreePreservationOrder                     Boolean?
+  uploadDefinitiveMapStatement                    Boolean?
+  supplementaryPlanningDocs                       Boolean?
+  uploadSupplementaryPlanningDocs                 Boolean?
+  affectsScheduledMonument                        Boolean?
+  gypsyTraveller                                  Boolean?
+  statutoryConsultees                             String?
+  statutoryConsultees_consultedBodiesDetails      String?
+  protectedSpecies                                Boolean?
+  publicRightOfWay                                Boolean?
+  areaOutstandingBeauty                           Boolean?
+  designatedSites                                 String?
+  designatedSites_otherDesignations               String?
+  screeningOpinion                                Boolean?
+  environmentalStatement                          Boolean?
+  environmentalImpactSchedule                     String?
+  uploadEnvironmentalStatement                    Boolean?
+  columnTwoThreshold                              Boolean?
+  sensitiveArea                                   String?
+  sensitiveArea_sensitiveAreaDetails              String?
+  uploadScreeningOpinion                          Boolean?
+  uploadScreeningDirection                        Boolean?
+  developmentDescription                          String?
+  requiresEnvironmentalStatement                  String?
+
+  // Fields below in confluence master doc but no corresponding question yet
+
+  // otherDesignationDetails         String?
+  // completedEnvironmentalStatement Boolean?
+  // appealSiteAccess                Boolean?
+  // appealSiteAccessDetails							String?
+  // lpaStatement                    String?
+  // lpaStatementDocuments           Boolean?
+  // lpaFinalComment                 Boolean?
+  // lpaFinalCommentDetails          String?
+  // lpaWitnesses                    Boolean?
+
+  // Uploaded documents and details linked to this appeal
+  SubmissionDocumentUpload   SubmissionDocumentUpload[]
+  SubmissionNeighbourAddress SubmissionNeighbourAddress[]
+}
+
+model SubmissionDocumentUpload {
+  questionnaireId            String                     @db.UniqueIdentifier
+  LPAQuestionnaireSubmission LPAQuestionnaireSubmission @relation(fields: [questionnaireId], references: [id])
+
+  id               String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
```

Comment:
picky but I'd expect id first in a schema, might just be me though

Reply:
Moved